tools/clang-format/package.json

```diff
@@ -1,9 +1,5 @@
 {
-  "name": "node-core-clang-format",
-  "version": "1.0.0",
-  "description": "Formatting C++ files for Node.js core",
-  "license": "MIT",
   "dependencies": {
-    "clang-format": "^1.8.0"
+    "node-core-clang-format": "^0.0.4"
```

Comment:
I assume this is your repository looking from https://www.npmjs.com/package/node-core-clang-format. I don't think we should adopt a fork which is closed source, without any readme etc.

Reply:
It's only temporary, I set it up in a heartbeat. Im going to draft this PR until I have time to continue working on it next week or the week after